### ISSUE TITLE
Has this project been abandoned?

### ISSUE THREAD

**Issue by MrPowers** [carlhuda/janus#708]
Thanks for making and maintaining this great project for so long.

I've been using Janus for years, ever since reading [Yehuda's blog](https://yehudakatz.com/2010/07/29/everyone-who-tried-to-convince-me-to-use-vim-was-wrong/).

Is this project going to be maintained anymore?  No judgement either way - I know maintaining a project like this is a ton of work.  Thanks again for making this project - it's what made me fall in love with Vim!

**Comment by kalbasit** [comment]
@MrPowers the project is in semi-maintenace state I'd say, and I'll be more than happy to accept pull requests, but as I don't use it myself anymore I won't be able to put any effort in maintaining it. I'm currently focusing my efforts on https://github.com/SoxinOS/soxin and one of my objectives on that project is to deliver a reproducible Vim/NeoVim distribution (among other software) with focus on programming language support, theme support etc across all software. I'll be announcing here soon once I have something to announce.

**Comment by MrPowers** [comment]
Awesome, thanks @kalbasit, appreciate the response.

I don't have time to maintain this either.  Maybe we could put out some messaging that we're looking for a maintainer?  There is a popular project and I'm sure someone would be willing to maintain it 🙏

**Comment by kalbasit** [comment]
Yes, that's my plan anyway @MrPowers it would be helpful if you could get the message out to your circles, and I'll do the same. I'll follow-up soon with a change to the README.

**Comment by omenking** [comment]
I still use Janus but NerdTree is the largest plain.
Couldn't get Fern to work as of yet.

I really like the defaults for this project, though just replicate with something modern will take lots of combing.

**Comment by MrPowers** [comment]
Just wanted to follow up and say I miss this project.

I am trying another Vim distribution now that's maintained and I just don't like it as much as I liked Janus.  Really miss this great project.

I know that some things don't last forever and I am grateful for the time I had with Janus.  It's what got me into Vim.  Highly encourage another hacker to build a janus-like project.

**Comment by omenking** [comment]
@MrPowers I still use it LOL.
